cluster/gce/windows/k8s-node-setup.psm1

```diff
@@ -51,7 +51,7 @@
 #  - Document functions using proper syntax:
 #    https://technet.microsoft.com/en-us/library/hh847834(v=wps.620).aspx
 
-$INFRA_CONTAINER = 'gcr.io/gke-release/pause-win:1.0.0'
+$INFRA_CONTAINER = 'mcr.microsoft.com/k8s/core/pause:1.2.0'
```

Comment:
@yliaog does `gke-release/pause-win:1.0.0` already include the port-forwarding support?

If so then is this change necessary?

Reply:
@yliaog do you know?

Reply:
It seems not:
```
PS C:\> wincat
wincat : The term 'wincat' is not recognized as the name of a cmdlet, function, script file, or operable program.
Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
At line:1 char:1
+ wincat
+ ~~~~~~
    + CategoryInfo          : ObjectNotFound: (wincat:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException
```

I got above from a `gcr.io/gke-release/pause-win:1.0.0` container

Reply:
It is in the image, just not in PATH in powershell for some reason.

I'll remove the image update. :)